addons/sale/views/sale_portal_templates.xml

```diff
@@ -211,20 +211,14 @@
                                     <h4 class="modal-title">Validate Order</h4>
                                     <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
                                 </header>
-                                <main class="modal-body" id="sign-dialog">
+                                <main class="modal-body pt-0" id="sign-dialog">
                                     <p>
-                                        <span>By signing this proposal, I agree to the following terms:</span>
-                                        <ul>
-                                            <li>
-                                                <span>Accepted on the behalf of:</span> <b t-field="sale_order.partner_id.commercial_partner_id"/>
-                                            </li>
-                                            <li>
-                                                <span>For an amount of:</span> <b data-id="total_amount" t-field="sale_order.amount_total"/>
-                                            </li>
-                                            <li t-if="sale_order.payment_term_id">
-                                                <span>With payment terms:</span> <b t-field="sale_order.payment_term_id.note"/>
-                                            </li>
-                                        </ul>
+                                        By signing, you confirm acceptance on behalf of
+                                        <b t-field="sale_order.partner_id.commercial_partner_id"/>
+                                        for the <b data-id="total_amount" t-field="sale_order.amount_total"/> quote
+                                        <span t-if="sale_order.payment_term_id">
```

Comment:
It's missing a dot at the end when `sale_order.payment_term_id` is falsy. Take care not to over-complexify the construction of the string as it may become hard to translate.